twitterFederation/services/comments/src/resolvers/mutation.ts

```diff
@@ -0,0 +1,17 @@
+import Comment from '../models/comment';
+
+export default {
+  addComment: async (_: any, data: any) => {
+    const comment = await Comment.create(data.comment);
+    return comment;
+  },
+  updateComment: async (_: any, data: any) => {
+    const { id, ...comment_data } = data.comment;
```

Comment:
You can do object destructuring  in the function parameters:
```suggestion
  updateComment: async (_: any, { comment: { id, ...comment_data } }) => {
```